tests/sanity_helpers.py

```diff
@@ -36,13 +40,14 @@ def health_check(self):
         )
         self.ceph_cluster.cluster_health_check(timeout=60)
 
-    def create_resources(self, pvc_factory, pod_factory, run_io=True):
+    def create_resources(self, pvc_factory, pod_factory, dc_pod_factory, run_io=True):
```

Comment:
why not replacing pod with DC?

Reply:
It would be good if we keep both as these 2 act's differently with their behavior